src/main/java/dev/lsegal/jenkins/codebuilder/CodeBuilderComputer.java

```diff
@@ -77,7 +78,7 @@ public void taskAccepted(Executor executor, Queue.Task task) {
   public void taskCompleted(Executor executor, Queue.Task task, long durationMS) {
     super.taskCompleted(executor, task, durationMS);
     LOGGER.info("[CodeBuilder]: [{}]: Task in job '{}' completed in {}ms", this, task.getFullDisplayName(), durationMS);
-    if (cloud.isTerminateAgent()) {
+    if (cloud != null && cloud.isTerminateAgent()) {

```

Comment:
Reduce  `if (cloud != null && cloud.isTerminateAgent())` to `if (cloud && cloud.isTerminateAgent())`

Reply:
I got the error below after trying to make this change.

`[ERROR] /mnt/c/Users/fgoncalves/Documents/repos/CI-CD-CodeBuildJenkinsPlugin/src/main/java/dev/lsegal/jenkins/codebuilder/CodeBuilderComputer.java:[80,15] bad operand types for binary operator '&&'
  first type:  dev.lsegal.jenkins.codebuilder.CodeBuilderCloud
  second type: boolean
[ERROR] /mnt/c/Users/fgoncalves/Documents/repos/CI-CD-CodeBuildJenkinsPlugin/src/main/java/dev/lsegal/jenkins/codebuilder/CodeBuilderComputer.java:[91,15] bad operand types for binary operator '&&'
  first type:  dev.lsegal.jenkins.codebuilder.CodeBuilderCloud
  second type: boolean`

Reply:
I'll leave it as is since the change causes an error.